src/core_deku/state.ml

```diff
@@ -97,9 +97,9 @@ let apply_user_operation t user_operation =
         ~address:to_invoke ~updated_contract:contract in
     Ok ({ ledger; contract_storage }, None)
 
-let apply_user_operation t user_operation =
-  match apply_user_operation t user_operation with
+let apply_user_operation t hash user_operation =
+  match apply_user_operation t hash user_operation with
   | Ok (t, receipt) -> (t, receipt)
   (* TODO: use this erros for something *)
   | Error (`Origination_error _ | `Invocation_error _) -> (t, None)
-  | Error `Not_enough_funds -> (t, None)
+  | Error (`Insufficient_funds | `Not_enough_funds) -> (t, None)
```

Comment:
Looks like we added an extra error here? `Insufficient` is synonym to `Not_enough` so we ought not have both.

Reply:
Sorry, I was tricked by my local environment mentioning the `this pattern is not exhaustive error`. Fixed.